tests/Feature/AbstractRequestTest.php

```diff
@@ -366,6 +398,12 @@ public function getLogFolder(): string
         } catch (\DomainException $exception) {
             self::assertSame('No log files have been saved by this instance.', $exception->getMessage());
         }
+        try {
```

Comment:
This also resolves a gripe CHowie and I found today, that `getLastLogFile` was returning the folder name if the request had never been called, instead of a nice clean exception like it's twin `getLastLogContents`